### ISSUE TITLE
display redundancy (genes /extensions)

### ISSUE THREAD

**Issue by ValWood** [pombase/website#343]
this
![redundant](https://cloud.githubusercontent.com/assets/7359272/26226746/f0b8369e-3c25-11e7-9710-410c4db483ea.jpg)

might be better as
blt1 during,during 
cdr1  during, during

etc

**Comment by ValWood** [comment]
Not critical, but probably quick. Can punt...

**Comment by ValWood** [comment]
remove milestone if not quick

**Comment by ValWood** [comment]
another e.g.

![collapse](https://cloud.githubusercontent.com/assets/7359272/26506298/eb562dfe-4243-11e7-9b6c-16848fce1ef7.jpg)

**Comment by kimrutherford** [comment]
I've made that change.  The code for compressing extensions in the summary only worked on extensions that had a gene as the range.  I've fixed it to work in the case where the range is a term.

Here's what your first example looks like now:
http://preview.pombase.org/term/GO%3A0071341

![screenshot from 2017-05-28 19-10-02](https://cloud.githubusercontent.com/assets/90474/26526824/8ac4a52c-43da-11e7-8eca-1dd3242b3391.png)
